src/makerom/rom_header.s

```diff
@@ -9,7 +9,11 @@
 /* 0x18 */ PADDING(8)
 /* 0x20 */ ROM_NAME("THE LEGEND OF ZELDA")
 /* 0x34 */ PADDING(7)
-/* 0x3B */ MEDIUM(CARTRIDGE)
+#if OOT_NTSC
+/* 0x3B */ MEDIUM(STORAGE_MEDIUM_CARTRIDGE_EXPANDABLE)
+#else
+/* 0x3B */ MEDIUM(STORAGE_MEDIUM_CARTRIDGE)
+#endif
 /* 0x3C */ GAME_ID("ZL")
-/* 0x3E */ REGION(PAL)
+/* 0x3E */ REGION(OOT_REGION)
```

Comment:
I'm not sure this is worth it over just using another `#if` condition here. Or maybe the makefile could define `OOT_REGION` to `REGION($(REGION))`? I'd rather maintain the token pasting since it doubles as macro argument validation.

Reply:
Discussed offline, I didn't know there were already existing REGION_US/REGION_JP/REGION_EU defines, and REGION_NATIVE which is essentially what OOT_REGION is. I split them out into include/region.h, used an `#if` in rom_header.s, and restored the token pasting